corehq/apps/reports/static/reports/js/config.dataTables.bootstrap.js

```diff
@@ -92,7 +92,7 @@ hqDefine("reports/js/config.dataTables.bootstrap", [
             }
             applyBootstrapMagic();
 
-            var dataTablesDom = "frt<'row dataTables_control'<'col-sm-5'il><'col-sm-7'p>>";
```

Comment:
whoa, what even is this syntax?

Reply:
beats me...i was quite relieved this was fixable just by adding a class to this element